os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -520,34 +522,42 @@ static int reloading_thread(int argc, char *argv[])
 		return ERROR;
 	}
 
-	/* argv[1] binary index for reloading */
-	int bin_idx = (int)atoi(argv[1]);
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
-	if (bin_idx == BM_CMNLIB_IDX) {
-		int ret;
+	int ret;
+	int bidx;
+	int bin_count = binary_manager_get_ucount();
 
-		/* Reload common library and all binaries */
-		char libname[CONFIG_NAME_MAX];
-		snprintf(libname, CONFIG_NAME_MAX, BINMGR_DEVNAME_FMT, BIN_PARTNUM(BM_CMNLIB_IDX, 0));
-		ret = load_binary(BM_CMNLIB_IDX, libname, NULL);
-		if (ret < 0) {
+	/* Unload all user binaries */
+	for (bidx = 1; bidx <= bin_count; bidx++) {
```

Comment:
If there is unloading sequence, such as user binary first, common last, please leave comment why.
If there is no, how about unloading the common binary also in here?

Reply:
I added comments.